### PR TITLE
Add show_stars attribute to hide/show significance stars

### LINKED ISSUE #28: Hide stars
Many journal articles no longer show stars for significance levels.

Here's a workaround:
```
stargazer.significance_levels([1e-999, 1e-999, 1e-999])
stargazer.append_notes(False)
```

### PR DESCRIPTION
**Pull request by MaxGhenis**
Fixed #28.

Aside: `append_notes` could have a more descriptive and consistent name, maybe `show_significance_note`?

### CODE REVIEW

File: stargazer/stargazer.py [hunk 1]
```diff
@@ -216,6 +217,10 @@ def append_notes(self, append):
         assert type(append) == bool, 'Please input True/False'
         self.notes_append = append
 
+    def show_stars(self, show):
+        assert type(show) == bool, 'Please input True/False'
+        self.show_stars = show
```
Comment (toobaz) [inline stargazer/stargazer.py:222]: Same comment as https://github.com/mwburke/stargazer/pull/55/files#r446540348 ... I don't think you tested this ;-)

**Comment by toobaz** [comment]
Apart from the comment on the implementation, I like the idea of this PR.

> Aside: `append_notes` could have a more descriptive and consistent name, maybe `show_significance_note`?

I agree

**Comment by MaxGhenis** [comment]
Removed the show_stars function and fixed merge conflicts

**Comment by toobaz** [comment]
Thanks @MaxGhenis !

**Pull request merged**